doc/nrf/ug_nrf91_features.rst

```diff
@@ -160,70 +160,90 @@ The :ref:`nrfxlib:nrf_modem` is released as an OS-independent binary library in
 The Modem library integration layer fulfills the integration requirements of the Modem library in |NCS|.
 For more information on the integration, see :ref:`nrf_modem_lib_readme`.
 
+.. _nrf9160_fota:
 
-.. _nrf9160_ug_band_lock:
+FOTA upgrades
+=============
 
-Band lock
-*********
+|fota_upgrades_def|
+FOTA upgrades can be used to apply delta patches to the :ref:`lte_modem` firmware, full :ref:`lte_modem` firmware upgrades, and to replace the upgradable bootloader or the application.
```

Comment:
FOTA upgrades can be managed through a comprehensive management portal on \`nRF Cloud\`\_, either fully hosted on nRF Cloud or accessible from a customer cloud using the \`nRF Cloud REST API\`\_.  FOTA upgrades can alternatively be hosted from a customer-developed cloud service; examples are provided for AWS and Azure.